content/_data/upgrades/2-302-1.adoc

```diff
@@ -0,0 +1,176 @@
+* `jenkins/jenkins:2.302.1-slim`
+
+Additional controller images and tags have been added for those users who need to remain on Java 8.
+The new Java 8 tags include:
+
+* `jenkins/jenkins:lts-jdk8`
+* `jenkins/jenkins:lts-alpine-jdk8`
+* `jenkins/jenkins:lts-centos7-jdk8`
+* `jenkins/jenkins:lts-slim-jdk8`
+* `jenkins/jenkins:2.302.1-jdk8`
+* `jenkins/jenkins:2.302.1-alpine-jdk8`
+* `jenkins/jenkins:2.302.1-centos7-jdk8`
+* `jenkins/jenkins:2.302.1-slim-jdk8`
+
+===== Inbound agent images
+
+Jenkins inbound agent images that do not specify a Java version now use Java 11 instead of Java 8.
+Jenkins inbound agent images with the following tags will use Java 11:
+
+* `jenkins/inbound-agent:alpine`
+* `jenkins/inbound-agent:latest`
+* `jenkins/inbound-agent:4.10-1`
+* `jenkins/inbound-agent:4.10-1-alpine`
+* `jenkins/inbound-agent:4.10-1-windowsservercore-ltsc2019`
+
+Additional inbound agent images and tags have been added for those users who need to remain on Java 8.
+The new Java 8 tags include:
+
+* `jenkins/inbound-agent:alpine-jdk8`
+* `jenkins/inbound-agent:latest-jdk8`
+* `jenkins/inbound-agent:4.10-1-jdk8`
+* `jenkins/inbound-agent:4.10-1-alpine-jdk8`
+* `jenkins/inbound-agent:4.10-1-windowsservercore-ltsc2019-jdk8`
+
+===== Outbound agent images
+
+Jenkins outbound agent images that do not specify a Java version now use Java 11 instead of Java 8.
+Jenkins outbound agent images with the following tags will use Java 11:
+
+* `jenkins/ssh-agent:alpine`
+* `jenkins/ssh-agent:latest`
+* `jenkins/ssh-agent:3.1.0`
+* `jenkins/ssh-agent:3.1.0-alpine`
+* `jenkins/ssh-agent:3.1.0-windowsservercore-ltsc2019`
+
+Additional outbound agent images and tags have been added for those users who need to remain on Java 8.
+The new Java 8 tags include:
+
+* `jenkins/ssh-agent:alpine-jdk8`
+* `jenkins/ssh-agent:latest-jdk8`
+* `jenkins/ssh-agent:3.1.0-jdk8`
+* `jenkins/ssh-agent:3.1.0-alpine-jdk8`
+* `jenkins/ssh-agent:3.1.0-windowsservercore-ltsc2019-jdk8`
+
+===== Agent images
+
+Jenkins agent images that do not specify a Java version now use Java 11 instead of Java 8.
+Jenkins agent images with the following tags will use Java 11:
+
+* `jenkins/agent:alpine`
+* `jenkins/agent:archlinux`
+* `jenkins/agent:latest`
+* `jenkins/agent:latest-archlinux`
+* `jenkins/agent:bullseye`
+* `jenkins/agent:4.10-1`
+* `jenkins/agent:4.10-1-alpine`
+* `jenkins/agent:4.10-1-windowsservercore-ltsc2019`
+
+Additional agent images and tags have been added for those users who need to remain on Java 8.
+The new Java 8 tags include:
+
+* `jenkins/agent:alpine-jdk8`
+* `jenkins/agent:archlinux-jdk8`
+* `jenkins/agent:latest-jdk8`
+* `jenkins/agent:latest-archlinux-jdk8`
+* `jenkins/agent:bullseye-jdk8`
+* `jenkins/agent:4.10-1-jdk8`
+* `jenkins/agent:4.10-1-alpine-jdk8`
+* `jenkins/agent:4.10-1-windowsservercore-ltsc2019-jdk8`
+
+==== Obsolete Docker images and tags no longer published
+
+Jenkins Docker images for new releases are not published for `centos`.
```

Comment:
For 2.306 there are [3 centos packages](https://hub.docker.com/r/jenkins/jenkins/tags?page=1&ordering=last_updated&name=306-centos) and this upgrade guide mentions `centos7` several times. Is this only for certain CentOS / JDK versions?

Reply:
it's centos8 which is tagged as centos on docker hub